<launch>

    <include file="$(find tof_vio)/launch/vio.launch"/>

    <node pkg="tof_vio" type="gt_publisher" name="gt" output="log">
        <remap from="/gt/vicon" to="/gt"/>
    </node>

    <node pkg="tof_vio" type="topic2file" name="odomfile" output="log">
        <remap from="~odom"        to="/icp_odom"/>
        <param name="filepath"     type="string"  value="$(find tof_vio)/result/icp+rd.txt"/>
    </node>
    <node pkg="tof_vio" type="topic2file" name="gtfile" output="log">
        <remap from="~odom"        to="/gt/odom_gt"/>
        <param name="filepath"     type="string"  value="$(find tof_vio)/result/gt.txt"/>
    </node>


    <node pkg="rosbag" type="play" name="rosbag" args="$(find tof_vio)/bag/handheld.bag"/>
    <!--<node pkg="rosbag" type="play" name="rosbag" args="$(find tof_vio)/bag/lab.bag"/>-->
    <!--<node pkg="rosbag" type="play" name="rosbag" args="$(find tof_vio)/bag/corridor.bag"/>-->

</launch>

